osltweaks.php

```diff
@@ -44,6 +44,24 @@ function osltweaks_civicrm_buildForm($formName, &$form) {
         }
       }
     }
+
+    // If contribution page id exist in $settings['member_redirect_contribution_pages'] array
+    if (array_key_exists($form->_id, ($settings['member_redirect_contribution_pages'] ?? array()))) {
+      // Get current contact id
+      $currentContactId = CRM_Core_Session::singleton()->getLoggedInContactID();
+      // Get memberships count base on the current contact id
+      $memberships = civicrm_api3('Membership', 'getcount', [
```

Comment:
@lucky091588 Since this is an integer count of memberships (not actual membership records), please name this variable $membershipCount.